GTG/gtk/editor/editor.py

```diff
@@ -52,6 +53,7 @@ def __init__(self,
         vmanager is the view manager
         thisisnew is True when a new task is created and opened
         '''
+        self.store = None
```

Comment:
You doing neither of the proposed alternatives. Please, define the store directly in .ui file